src/otx/algo/detection/heads/rtmdet_head.py

```diff
@@ -452,3 +548,175 @@ def get_anchors(
             multi_level_flags = self.prior_generator.valid_flags(featmap_sizes, img_meta["img_shape"], device)
             valid_flag_list.append(multi_level_flags)
         return anchor_list, valid_flag_list
+
+
+class RTMDetSepBNHead(RTMDetHead):
+    """RTMDetHead with separated BN layers and shared conv layers.
+
+    Args:
+        num_classes (int): Number of categories excluding the background
+            category.
+        in_channels (int): Number of channels in the input feature map.
+        share_conv (bool): Whether to share conv layers between stages.
+            Defaults to True.
+        use_depthwise (bool): Whether to use depthwise separable convolution in
+            head. Defaults to False.
+        norm_cfg (:obj:`ConfigDict` or dict)): Config dict for normalization
+            layer. Defaults to dict(type='BN', momentum=0.03, eps=0.001).
+        act_cfg (:obj:`ConfigDict` or dict)): Config dict for activation layer.
+            Defaults to dict(type='SiLU').
+        pred_kernel_size (int): Kernel size of prediction layer. Defaults to 1.
+    """
+
+    def __init__(
+        self,
+        num_classes: int,
+        in_channels: int,
+        share_conv: bool = True,
+        use_depthwise: bool = False,
+        norm_cfg: dict | None = None,
+        act_cfg: dict | None = None,
+        pred_kernel_size: int = 1,
+        exp_on_reg: bool = False,
```

Comment:
Could you add docstring for this argument?

Reply:
Done at https://github.com/openvinotoolkit/training_extensions/pull/3542/commits/589b361727fb74be22273eb9aaa6dab48133760b.